node/src/components/chainspec_loader.rs

```diff
@@ -301,6 +304,12 @@ impl ChainspecLoader {
         self.reactor_exit
     }
 
+    /// Returns whether the current node instance is started immediately after an upgrade –
+    /// i.e. whether the last/highest block stored is a block that trigerred the upgrade.
```

Comment:
```suggestion
    /// i.e. whether the last/highest block stored is a block that triggered the upgrade.
```

Reply:
Fixed in [`b1f152a` (#1757)](https://github.com/casper-network/casper-node/pull/1757/commits/b1f152a9a6db463faf34ae4b3804b004a8a9f017)